frontend/src/components/core/Container.tsx

```diff
@@ -10,6 +10,27 @@ const createContainer = (Container: any, props: MyProps) => {
   return <Container>{children}</Container>;
 };
 
+export const FlexContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  max-height: 100vh;
+  overflow: hidden;
+`;
+
+export const FlexInfoBar = styled.div`
+  flex: 1 0 100%;
+  padding: 1rem;
+  height: 30px;
+  line-height: 30px;
+  text-align: center;
+`;
+
+export const FlexPanel = styled.div`
+  flex: 1;
+  padding: 1rem;
+  background-color: #e3e3e3;
+`;
```

Comment:
I'm actually not sure if these two should go in the container file; at the same time, I think it does make sense to have the flex container in this file, and these three components should be grouped together. So idk really what the best situation is here. Can FlexContainer be reused in other classes (I'm not too familiar with flex)? If so, then maybe keep that in here and over flex info bar/panel back to Game.tsx. If it's pretty specific to the game page/only is useful with the info bar and panel, then I think maybe it can actually be put back in the Game.tsx file (maybe renaming it to Content that way it doesn't get flagged for refactoring later on).

I'm not really familiar or sure on what the best route forward is with this, so leaving it for now should be also fine.